projects/1.21/assets/psi/psi/lang/zh_cn.json

```
@@ -0,0 +1,1123 @@
+  "psi.programmer_reference10": "设置注释：\u00a7b%1$s-D",
+  "psi.programmer_reference11": "切换焦点：\u00a7bTab",
+  "psi.programmer_reference12": "显示提示框：\u00a7bAlt",
+  "psi.programmer_reference13": "移动光标：\u00a7b方向键",
+  "psi.programmer_reference14": "上传至Imgur：\u00a7b%1$s-Shift-Alt-G",
+  "psi.programmer_reference15": "分享至Reddit：\u00a7b%1$s-Shift-Alt-R",
+  "psi.programmer_reference16": "",
+  "psi.programmer_reference17": "\u00a7b高级搜索",
+  "psi.programmer_reference18": "输入类型：\u00a7bin:(类型)",
+  "psi.programmer_reference19": "输出类型：\u00a7bout:(类型)",
+  "psi.programmer_reference20": "名称前缀：\u00a7b(前缀)_",
+  "psi.programmer_reference21": "名称后缀：\u00a7b_(后缀)",
+  "psi.programmer_reference22": "来源模组：\u00a7b@(模组名)",
+  "psi.spellpiece.selector_caster": "选择符：施术者",
+  "psi.spellpiece.selector_caster.desc": "获取术式的施展者",
+  "psi.spellpiece.selector_focal_point": "选择符：焦点",
+  "psi.spellpiece.selector_focal_point.desc": "获取术式的焦点实体",
+  "psi.spellpiece.selector_nearby_items": "选择符：附近物品",
+  "psi.spellpiece.selector_nearby_items.desc": "获取指定位置附近的物品",
+  "psi.spellpiece.selector_nearby_living": "选择符：附近活体",
+  "psi.spellpiece.selector_nearby_living.desc": "获取指定位置附近的活物",
+  "psi.spellpiece.selector_nearby_enemies": "选择符：附近敌人",
+  "psi.spellpiece.selector_nearby_enemies.desc": "获取指定位置附近的敌人",
+  "psi.spellpiece.selector_nearby_animals": "选择符：附近动物",
+  "psi.spellpiece.selector_nearby_animals.desc": "获取指定位置附近的动物",
+  "psi.spellpiece.selector_nearby_projectiles": "选择符：附近弹射物",
+  "psi.spellpiece.selector_nearby_projectiles.desc": "获取指定位置附近的弹射物",
+  "psi.spellpiece.selector_nearby_smeltables": "选择符：附近可烧炼",
+  "psi.spellpiece.selector_nearby_smeltables.desc": "获取指定位置附近的可烧炼物品",
+  "psi.spellpiece.selector_loopcast_index": "选择符：循环施术索引",
+  "psi.spellpiece.selector_loopcast_index.desc": "获取循环型子弹已循环的次数",
+  "psi.spellpiece.selector_block_broken": "选择符：被破坏方块",
+  "psi.spellpiece.selector_block_broken.desc": "获取工具所破坏的方块",
+  "psi.spellpiece.selector_block_side_broken": "选择符：方块破坏面",
+  "psi.spellpiece.selector_block_side_broken.desc": "获取工具破坏方块时选定的面",
+  "psi.spellpiece.selector_attack_target": "选择符：攻击目标",
+  "psi.spellpiece.selector_attack_target.desc": "获取施术者所攻击的实体",
+  "psi.spellpiece.selector_sneak_status": "选择符：潜行状态",
+  "psi.spellpiece.selector_sneak_status.desc": "获取施术者在潜行与否（未潜行为1，潜行时为0）",
+  "psi.spellpiece.selector_time": "选择符：时间",
+  "psi.spellpiece.selector_time.desc": "获取CAD的内部时间（以刻计）",
+  "psi.spellpiece.selector_attacker": "选择符：攻击者",
+  "psi.spellpiece.selector_attacker.desc": "获取攻击施术者的实体",
+  "psi.spellpiece.selector_damage_taken": "选择符：所受伤害",
+  "psi.spellpiece.selector_damage_taken.desc": "获取施术者受到的伤害量",
+  "psi.spellpiece.selector_ruler_vector": "选择符：矢量尺",
+  "psi.spellpiece.selector_ruler_vector.desc": "获取矢量尺中设定的矢量",
+  "psi.spellpiece.selector_item_presence": "选择符：物品实在性",
+  "psi.spellpiece.selector_item_presence.desc": "获取物品栏中物品组的物品数量",
+  "psi.spellpiece.selector_block_presence": "选择符：方块实在性",
+  "psi.spellpiece.selector_block_presence.desc": "获取方块的实在性和固体性",

```

```suggestion
  "psi.spellpiece.selector_item_presence": "选择符：物品存在",
  "psi.spellpiece.selector_item_presence.desc": "获取物品栏中单个物品堆叠的物品数量",
  "psi.spellpiece.selector_block_presence": "选择符：方块存在",
  "psi.spellpiece.selector_block_presence.desc": "获取方块的存在和固体性",
```
`实在性`有点过于抽象了；`物品堆叠`作为专有名词感觉还是得整一下